temoa/data_io/hybrid_loader.py

```
@@ -24,7 +24,7 @@
 from collections.abc import Sequence
 from logging import getLogger
 from sqlite3 import Connection, Cursor, OperationalError
-from typing import Any
+from typing import Dict, List, Optional, Tuple
```

Ruff (0.14.0)

`typing.Dict` is deprecated, use `dict` instead

(UP035)